client/README.md

```
@@ -1,96 +1,73 @@
-This is a [Next.js](https://nextjs.org) project bootstrapped with [`create-next-app`](https://nextjs.org/docs/app/api-reference/cli/create-next-app).
+# Welcome to your Lovable project
 
-## Getting Started
+## Project info
 
-First, run the development server:
+**URL**: https://lovable.dev/projects/3a0326fb-1b99-4fe6-8531-ccd27b73381c
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)